docs/2.0/admin-guide.md

```diff
@@ -1076,7 +1077,12 @@ to run in a highly available fashion.
     auth server in `auth_servers` section of Teleport configuration.
 
 #### Run multiple instances of Teleport Proxy 
-  In this case, if one of the proxies becomes unavailalbe, you can use another one. The proxies are stateless so it's trivial to do.
+
+  The Teleport Proxy is stateless which makes running multiple instances
+  trivial. Configure your load balancer to forward ports `3023` and
```

Comment:
these ports are configurable. Also you should point to the ports table in the same doc for reference.